README.md

```diff
@@ -94,6 +94,37 @@ spec:
 
 Accessible with the url: http://traefik.localhost/dashboard/
 
+## Upgrading
+
+```bash
+# Update repository
+helm repo update
+# See current Chart & Traefik version
+helm search repo traefik/traefik
+# Upgrade Traefik
+helm upgrade traefik traefik/traefik
+```
+
+New major version indicates that there is an incompatible breaking change.
+
+### Upgrading CRDs
+
+With Helm v3, CRDs created by this chart can not be updated, cf the [Helm Documentation on CRDs](https://helm.sh/docs/chart_best_practices/custom_resource_definitions). Please read carefully release notes of this chart before upgrading CRDs.
+
+```bash
+kubectl apply --server-side --force-conflicts -f https://raw.githubusercontent.com/traefik/traefik-helm-chart/master/traefik/crds/ingressroute.yaml
```

Comment:
We can condense by adding a `kustomization.yaml` in `traefik/crds` and them making the `kubectl apply` like this:

```bash
kubectl apply --server-side --force-conflicts -k https://github.com/traefik/traefik-helm-chart/traefik/crds/
```

Reply:
Yes, it seems better. 
I've added the kustomization.yaml.

Reply:
This prevents picking a precise version though...